src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration monitorInterval="60">
+
+	<Properties>
+		<property name="logRepo">/var/log/</property>
```

Comment:
Remove ending / to fix logging error in travis

Reply:
done